Makefile

```diff
@@ -30,8 +30,11 @@ else
 	GO_BUILD_TAGS := assets,noasm
 endif
 
-GO_TEST_ARGS := -tags '$(GO_TEST_TAGS)'
-GO_BUILD_ARGS := -tags '$(GO_BUILD_TAGS)'
+# Tags used for builds and tests on all architectures
+COMMON_TAGS := sqlite_foreign_keys,sqlite_json
```

Comment:
These should get added to the `.goreleaser.yml` too (it bypasses the `Makefile`)

Reply:
Made the updates to the gorelease file...let me know if you think of anything else (docker?) that would need to be updated.

Reply:
Docker should be ok, it pulls the already-built binaries.

The Homebrew formula will need to be updated when we release 2.1